src/pages/resultsView/comparison/ResultsViewComparisonStore.ts

```diff
@@ -200,4 +194,57 @@ export default class ResultsViewComparisonStore extends ComparisonStore {
     public get studies() {
         return this.resultsViewStore.studies;
     }
+
+    public get studyIds() {
+        return this.resultsViewStore.studyIds;
+    }
+
+    readonly customDriverAnnotationProfileIds = remoteData<string[]>(
+        {
+            await: () => [this.resultsViewStore.molecularProfilesInStudies],
+            invoke: async () => {
+                return _(
+                    this.resultsViewStore.molecularProfilesInStudies.result
+                )
+                    .filter(
+                        (profile: MolecularProfile) =>
+                            // discrete CNA's
+                            (profile.molecularAlterationType ===
+                                AlterationTypeConstants.COPY_NUMBER_ALTERATION &&
+                                profile.datatype === 'DISCRETE') ||
```

Comment:
is there a constant for this anywhere?

Reply:
Yes, updated to `DataTypeConstants.DISCRETE`